supplements/candlekeep-mysteries/source.xml

```diff
@@ -15,8 +15,8 @@
 		</description>
```

Comment:
from the last review
> also maybe it's preferable to add this line at the top of the description
> ```xml
> <h5 style="margin-top:-1em">GREAT BOOKS HIDE THEIR SECRETS WELL</h5>
> ```
> since it's a tagline on the website now.